Documentation/Fluid/ViewHelper/If.rst

```diff
@@ -111,3 +111,19 @@ Simple examples
 If there are no `f:then` or `f:else` ViewHelpers in use, the content between the `f:if` ViewHelpers will only be output if 
 the comparison returns a positive result. If you need to use the full if-then-else construct, then you'll have to 
 use the `f:then` / `f:else` ViewHelpers.
+
+Inline notation
+---------------
+You can also use the ViewHelper with an inline notation.
+
+::
+  {f:if(condition: '{numberVar} == 1', then: 'active', else: 'disable')}
+  {f:if(condition: varName, then: 'varName is set')}
```

Comment:
I would change this to make it more clear:  {f:if(condition: varName, then: 'varName evaluates to true')}. 

Example:
$varName=false;

- isset($varName); : returns true
- f:if(condition: varName : will be false